util/logutil/log.go

```diff
@@ -111,17 +109,64 @@ func InitLogger(cfg *LogConfig) error {
 		return errors.Trace(err)
 	}
 
-	// init logger for grpc debugging
+	err = initGRPCLogger(cfg)
+	if err != nil {
+		return errors.Trace(err)
+	}
+
+	return nil
+}
+
+func initGRPCLogger(cfg *LogConfig) error {
+
+	config := deepcopyLogConfig(&cfg.Config)
+	// hack: force stdout
+	config.File.Filename = ""
 	if len(os.Getenv("GRPC_DEBUG")) > 0 {
-		// more information for verbosity: https://github.com/google/glog#verbose-logging
-		gzap.ReplaceGrpcLoggerV2WithVerbosity(gl, GRPCLogDebugVerbosity)
+		config.Level = "debug"
```

Comment:
how about:

```golang
config := *cfg.Config
config.Level = "debug"
```

In this way we can make sure that `cfg` is not modified as well.